index.js

```diff
@@ -45,52 +50,85 @@ function getLatestData(caseData) {
     try {
-      var cases = data.cases;
-      var deaths = data.deaths;
-      var recovered = data.recovered;
+      var cases = (data.cases).toLocaleString('en');
+      var deaths = (data.deaths).toLocaleString('en');
+      var recovered = (data.recovered).toLocaleString('en');
       var dateLastUpdated = moment(data.updated).format('MMMM D, YYYY [at] h:mm a');
       const message = `As of ${dateLastUpdated}, the current worldwide COVID-19 numbers are:\n\n`
         + `Total Cases: ${cases}\n`
         + `Deaths: ${deaths}\n`
         + `Recovered: ${recovered}\n`
-        + `Active Cases: ${cases - deaths - recovered}`
+        + `Active Cases: ${cases - deaths - recovered}`;
 
-      bot.createMessage(channelId, message)
+      bot.createMessage(channelId, message);
     } catch(error) {
       bot.createMessage(channelId, 'My apologies. I wasn\'t able to get the worldwide numbers.');
     }
   }
-  else {
-
-    const { data } = await axios.get(`https://corona.lmao.ninja/countries/${query}`);
-      try {
-        var country = query == 'us' || query == 'usa' || query == 'uk' ? 'the '.concat(v.upperCase(query)) : v.titleCase(query);
-        var cases = data.cases;
-        var todayCases = data.todayCases;
-        var deaths = data.deaths;
-        var todayDeaths = data.todayDeaths;
-        var recovered = data.recovered;
-        var active = data.active;
-        var critical = data.critical;
-        var casesPer1M = data.casesPerOneMillion;
-        const message = `As of the latest update, the current COVID-19 numbers in ${country} are:\n\n`
-          + `Total Cases: ${cases}\n`
-          + `Deaths: ${deaths}\n`
-          + `Critical Condition: ${critical}\n`
-          + `Recovered: ${recovered}\n`
-          + `Active Cases: ${active}\n`
-          + `Cases Per Million: ${casesPer1M}\n\n`
-          + `New Cases Today: ${todayCases}\n`
-          + `New Deaths Today: ${todayDeaths}`
-
-        bot.createMessage(channelId, message)
-      } catch(error) {
-        bot.createMessage(channelId, `My apologies. I wasn\'t able to get the numbers for ${query}.`);
+  else if (query.length == 1) {
+    const { data } = await axios.get(`https://corona.lmao.ninja/countries/${v.lowerCase(query[0])}`);
+    try {
+      if (data == 'Country not found') {
+        throw new Exception(data);
       }
+      var country = query[0] == 'us' || query[0] == 'usa' ? 'the United States' : (query[0] == 'uk' ? 'the United Kingdom' : v.upperCase(query[0]));
+      var cases = (data.cases).toLocaleString('en');
+      var todayCases = (data.todayCases).toLocaleString('en');
+      var deaths = (data.deaths).toLocaleString('en');
+      var todayDeaths = (data.todayDeaths).toLocaleString('en');
+      var recovered = (data.recovered).toLocaleString('en');
+      var active = (data.active).toLocaleString('en');
+      var critical = (data.critical).toLocaleString('en');
+      var casesPer1M = (data.casesPerOneMillion).toLocaleString('en');
```

Comment:
As per comments above. Also added a `toLowerCase()` so a user can put in whatever they want case-wise.

```suggestion
      let country;
      if (query[0].toLowerCase() === 'us' || query[0].toLowerCase() === 'usa') {
        country = 'the United States';
      } else if(query[0].toLowerCase() == 'uk') {
        country = 'the United Kingdom';
      } else {
        country = v.upperCase(query[0]));
      }
      const {
        cases,
        todayCases,
        deaths,
        todayDeaths,
        recovered,
        active,
        critical,
        casesPerOneMillion: casesPer1M,
      } = data;
```